### PR TITLE
Project service

### PR DESCRIPTION
**Pull request by martazpk**
Project.class- changed constructor (pass Dto), projectService-changed add method because constructor of Project was changed

### CODE REVIEW

**Comment by smalaca** [comment]
I assume only the last commit (Project.class- changed constructor (pass Dto), projectService-change[...]) should be considered as a one to merge.
What you can do?
1. Ctrl + Shift + A
2. Type "Fetch"
3. Enter
4. Checkout new branch from master
5. Cherry pick your last commit (if you don't know how try to find phare "IntelliJ cherry pick" and in case of no success just write to me :)
6. Push and create new branch.
7. Open new PR and close this one 😄

File: .idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (smalaca) [inline .idea/compiler.xml]: remove this from PR.

File: .idea/misc.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (smalaca) [inline .idea/misc.xml]: remove this from PR.

File: .idea/modules.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (smalaca) [inline .idea/modules.xml]: remove this from PR.

File: .idea/vcs.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (smalaca) [inline .idea/vcs.xml]: remove this from PR.

File: src/main/java/pl/sda/jira/MyAwesomeDependency.java [hunk 5]
```diff
@@ -0,0 +1,10 @@
+package pl.sda.jira;
+
+import org.springframework.stereotype.Component;
+
+@Component
```
Comment (smalaca) [inline src/main/java/pl/sda/jira/MyAwesomeDependency.java]: this is from my example - should not be a part of the PR

File: src/main/java/pl/sda/jira/TemplateApplication.java [hunk 6]
```diff
@@ -0,0 +1,11 @@
+package pl.sda.jira;
```
Comment (smalaca) [inline src/main/java/pl/sda/jira/TemplateApplication.java]: this is from my example - should not be a part of the PR

File: src/main/java/pl/sda/jira/TemplateController.java [hunk 7]
```diff
@@ -0,0 +1,58 @@
+package pl.sda.jira;
```
Comment (smalaca) [inline src/main/java/pl/sda/jira/TemplateController.java]: this is from my example - should not be a part of the PR

File: src/main/java/pl/sda/jira/TemplateDto.java [hunk 8]
```diff
@@ -0,0 +1,31 @@
+package pl.sda.jira;
```
Comment (smalaca) [inline src/main/java/pl/sda/jira/TemplateDto.java]: this is from my example - should not be a part of the PR

File: src/test/java/pl/sda/jira/TemplateControllerTest.java [hunk 9]
```diff
@@ -0,0 +1,93 @@
+package pl.sda.jira;
```
Comment (smalaca) [inline src/test/java/pl/sda/jira/TemplateControllerTest.java]: this is from my example - should not be a part of the PR

**Comment by martazpk** [comment]
failed

**Pull request closed without merging**